projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.html

```diff
@@ -72,6 +72,20 @@ <h1>{{ 'Barcode' | translate }} {{ record.metadata.barcode }}</h1>
       <dd class="col-sm-7 col-md-8 mb-0">
         <admin-item-availability [item]="record"></admin-item-availability>
       </dd>
+      <!-- ACQUISITION -->
+      <dt class="col-sm-3 offset-sm-2 offset-md-0">
+        {{ 'New acquisition' | translate }}:
+      </dt>
+      <dd class="col-sm-7 col-md-8 mb-0">
+        <i class="fa"
+           [ngClass]="(record.metadata.acquisition_date) ? 'fa-check' : 'fa-times'"
+           [ngStyle]="{'color': (record.metadata.acquisition_date) ? 'green' : 'red'}"
```

Comment:
Using a class instead a style: `text-danger` and `text-success`

Reply:
done